Common/Utils/test/testBoostSerializer.cxx

```diff
@@ -26,60 +26,60 @@ using namespace o2::utils;
 
 BOOST_AUTO_TEST_SUITE(testDPLSerializer)
 
-BOOST_AUTO_TEST_CASE(testTrivialTypeVect)
-{
-  using contType = std::vector<int>;
+// BOOST_AUTO_TEST_CASE(testTrivialTypeVect)
+// {
+//   using contType = std::vector<int>;
```

Comment:
what's the reason for commenting all these tests? Are they going to be activated again, I would delete them otherwise.

Reply:
They have been commented because that particular feature was causing the issues with your ROOT serialization. Hopefully they'll be restored at a given point: boost can serialize trivial types, it's just something not under my control using boost instead of ROOT for serialization of ROOT classes' trivial data members.

I can add this explanation around the commented code block.

Reply:
good, then lets keep the commented code with some explanation